### DISCUSSION TITLE
Zoom link (with password) not working

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#1495] [Q&A]
This Google Sheet includes (2) Zoom meetings.
One is working, but the other isn't.
https://docs.google.com/spreadsheets/d/1AEjqCJqoHYTLPtg7pElBTDsWSWKC_Uy-dYp0qTD6qS4/edit?usp=sharing

The link that was provided includes a Password and  doesn't appear in the District's meeting list:
https://us02web.zoom.us/j/84352241003?pwd=Vm82TDlod0pITG5hVVNxOE9lZEVjUT09#

Here is the website, currently in staging:
https://custom-memories.com/website_1f23693b

Any help would be appreciated.
In fellowship,
Scott F., Area 59 web servant

**Comment by joshreisner** [comment]
yes `https://us02web.zoom.us/j/87669658243 Meeting ID: 876 6965 8243` is not a valid zoom URL - if you can move the `Meeting ID: 876 6965 8243` part to conference URL notes that should fix it

**Comment by fritzisa** [comment]
**Thanks for the guidance, Josh.**

That link works.
It's the second one (that includes a password) that doesn't appear in the meeting list.
Based on your comment, I'll revise both links, moving the Meeting ID and PW to the Conference URL Notes column.

Zoom URL:
https://us02web.zoom.us/j/84352241003

Move to Conference URL NOtes column:
pwd=Vm82TDlod0pITG5hVVNxOE9lZEVjUT09#

Thanks for your continued support!
I appreciate your help.

**Scott Fritzinger**  |  Area 59 Web Servant  |  (610) 762-7792

**Reply by joshreisner** [reply]
no - don't remove the `pwd=` -- it should definitely work with that in the URL -- removing it will make it hard for users

maybe you just need to take out the `#`? sort of surprised that's true tbh but i'm not sure where the error is exactly

are you importing the google sheet into TSML through sheets.code4recovery.org? maybe there is some step breaking down somewhere

**Reply by fritzisa** [reply]
OK, I put it back, so the Zoom link is:
https://us02web.zoom.us/j/84352241003?pwd=Vm82TDlod0pITG5hVVNxOE9lZEVjUT09#

This is the Google Sheet:
https://docs.google.com/spreadsheets/d/1AEjqCJqoHYTLPtg7pElBTDsWSWKC_Uy-dYp0qTD6qS4/edit?usp=sharing

Yes, I paste that link into the Google Sheets Importer, which generates this JSON:
https://sheets.code4recovery.org/storage/1AEjqCJqoHYTLPtg7pElBTDsWSWKC_Uy-dYp0qTD6qS4.json

**Reply by fritzisa** [reply]
I also noticed that only (47) meetings show up when I import the JSON, which means (3) meetings are being left out. (?)

**Reply by joshreisner** [reply]
when i refresh the feed i see some errors about missing ids: https://sheets.code4recovery.org/1AEjqCJqoHYTLPtg7pElBTDsWSWKC_Uy-dYp0qTD6qS4

i'm not seeing the link in your sheet

**Reply by fritzisa** [reply]
I had to add it back (sorry):
https://docs.google.com/spreadsheets/d/1AEjqCJqoHYTLPtg7pElBTDsWSWKC_Uy-dYp0qTD6qS4/edit?usp=sharing

After importing into Google Sheets Importer:
https://sheets.code4recovery.org/storage/1AEjqCJqoHYTLPtg7pElBTDsWSWKC_Uy-dYp0qTD6qS4.json

District 48 must have added meetings in Rows 21, 24, and 47, which have caused the "(3) Rows with empty ID or slug" message.

I don't know where that data comes from. (?)

**Reply by joshreisner** [reply]
you've got a `slug` column in column `N` and an `id` column in column `R` - you only need one of these, and they all have to have unique values

you dont need columns `M`, `O`, or `P` - they contain data that won't be imported

**Reply by fritzisa** [reply]
I removed the unneeded columns.
Where does the "slug data" come from?
And the Updated column says "2024-07-21 17:41:52" in each row?

**Reply by joshreisner** [reply]
i'm not sure where it originally got added from, but going forward you (or whoever is entering data) should make sure there's a unique, nonempty value in each row.

if the updated column is not accurate i would remove it…

**Reply by fritzisa** [reply]
OK, I'll create my own unique slug for those three meetings (rows) and delete the Updated column.

Area 59 only has three districts that have chosen to use Google Sheets, and it's been relatively simple except for this new District.

**Thanks for all your help.**
